pkg/server/server.go

```diff
@@ -784,6 +785,10 @@ func (api *APIServer) httpRestoreHandler(w http.ResponseWriter, r *http.Request)
 		tablePattern = tp[0]
 		fullCommand = fmt.Sprintf("%s --tables=\"%s\"", fullCommand, tablePattern)
 	}
+	if databaseMapping, exist := query["databaseMapping"]; exist {
```

Comment:
please use the same name as config
`restore_database_mapping` looks more mental we don't use "camelCase" in other query string parameters as you see